tests/bluetooth/controller/common/src/helper_util.c

```diff
@@ -293,14 +293,15 @@ uint16_t event_counter(struct ll_conn *conn)
 
 	/* If event_counter is called inside an event_prepare()/event_done() pair
 	 * return the current event counter value (i.e. -1);
-	 * otherwise return the next event counter value */
+	 * otherwise return the next event counter value
+	 */
 	if (event_active)
 		event_counter--;
 
 	return event_counter;
 }
 
-void lt_tx_real(const char *file, uint32_t line, helper_pdu_opcode_t opcode, struct ll_conn *conn,
+void lt_tx_real(const char *file, uint32_t line, enum helper_pdu_opcode opcode, struct ll_conn *conn,
```

Comment:
line is now over 100chars
```suggestion
void lt_tx_real(const char *file, uint32_t line, enum helper_pdu_opcode opcode, 
                       struct ll_conn *conn,
```

Reply:
Doh!